src/runtime/audience-action-flow-test.js

```diff
@@ -1339,69 +1339,159 @@ describes.realWin('AudienceActionIframeFlow', (env) => {
     dialogManagerMock.verify();
   });
 
-  it('opens dialog with closeOnBackgroundClick=false by default', async () => {
-    const audienceActionFlow = new AudienceActionIframeFlow(runtime, {
-      action: 'TYPE_REGISTRATION_WALL',
-      configurationId: 'configId',
-      onCancel: onCancelSpy,
-      autoPromptType: AutoPromptType.SUBSCRIPTION,
+  describe('closeOnBackgroundClick experiment active', () => {
+    after(() => {
+      dialogManagerMock.verify();
     });
-    dialogManagerMock
-      .expects('openView')
-      .withExactArgs(
-        sandbox.match.any,
-        false,
-        sandbox.match({
-          closeOnBackgroundClick: false,
-        })
-      )
-      .once();
-    await audienceActionFlow.start();
-    dialogManagerMock.verify();
-  });
 
-  it('opens dialog with closeOnBackgroundClick=false when isClosable=false', async () => {
-    const audienceActionFlow = new AudienceActionIframeFlow(runtime, {
-      action: 'TYPE_REGISTRATION_WALL',
-      configurationId: 'configId',
-      onCancel: onCancelSpy,
-      autoPromptType: AutoPromptType.SUBSCRIPTION,
-      isClosable: false,
+    it('opens dialog with closeOnBackgroundClick=false by default', async () => {
+      sandbox
+        .stub(runtime.entitlementsManager(), 'isExperimentEnabled')
+        .callsFake(() => Promise.resolve(true));
```

Comment:
This can be moved to a `before` block for the description since this is common setup for these cases.